src/index.html

```diff
@@ -7,6 +7,42 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="container">
+      <div class="calendar">
+        <div class="calendar__wed">
```

Comment:
For what do you need so much containers ?

Reply:
done see v2

Reply:
done see 'v2'